openpower/configs/hostboot/witherspoon.config

```diff
@@ -1,48 +1,46 @@
 # The Serial Flash Controller is the AST2400 BMC.
-set   SFC_IS_AST2400
+set   SFC_IS_AST2500
+unset SFC_IS_AST2400
 unset   BMC_DOES_SFC_INIT
 unset SFC_IS_IBM_DPSS
 set   ALLOW_MICRON_PNOR
 set   ALLOW_MACRONIX_PNOR
 
 # VPD options.
-unset MVPD_READ_FROM_HW
-unset MVPD_WRITE_TO_HW
+set MVPD_READ_FROM_HW
+set MVPD_WRITE_TO_HW
 set MVPD_READ_FROM_PNOR
 set MVPD_WRITE_TO_PNOR
-unset DJVPD_READ_FROM_HW
+set DJVPD_READ_FROM_HW
 unset DJVPD_WRITE_TO_HW
 set DJVPD_READ_FROM_PNOR
 set DJVPD_WRITE_TO_PNOR
-unset CVPD_READ_FROM_HW
-unset CVPD_WRITE_TO_HW
+set CVPD_READ_FROM_HW
```

Comment:
This is, on the surface, kind of odd to see for Witherspoon.  There is no Centaur and thus no CVPD for Witherspoon.

I suspect there is some odd arrangement between CVPD and MVPD code though that requires this.